HeterogeneousCore/SonicTriton/plugins/BuildFile.xml

```diff
@@ -0,0 +1,11 @@
+<library   name="HeterogeneousCoreSonicTritonPlugins_converters" file="converters/*.cc">
```

Comment:
recommended style is now to put only one space between xml attributes (everywhere)